test/rule-matches/landmark-unique-matches.js

```diff
@@ -116,73 +120,28 @@ describe('landmark-unique-matches', function () {
           axeFixtureSetup(
             '<' + elementType + '>an element</' + elementType + '>'
           );
-          var node = fixture.querySelector(elementType);
-          var virtualNode = axe.utils.getNodeFromTree(axe._tree[0], node);
+          const node = fixture.querySelector(elementType);
+          const virtualNode = axe.utils.getNodeFromTree(axe._tree[0], node);
           assert.isTrue(rule.matches(node, virtualNode));
         }
       );
     });
   });
 
-  describe('aside should not match when scoped to a sectioning content element unless it has an accessible name', function () {
```

Comment:
You seem to have accidentally reverted https://github.com/dequelabs/axe-core/pull/4469/files#diff-0ef6a2f04e39c405fcbf972fe657cad972110dc05e1a7d757d5a171e4a69140a